README.md

```diff
@@ -42,24 +42,96 @@ from great_expectations_provider.operators.great_expectations_bigquery import Gr
 
-Note that to make these operators work, you will need to change the value of [`enable_xcom_pickling`](https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg#L181) to `true` in your airflow.cfg.
+The example DAG can be exercised in one of two ways:
 
-These examples can be tested in one of two ways:
-
-**With the open-source Astro CLI:**
+**With the open-source Astro CLI (recommended):**
 1. Initialize a project with the [Astro CLI](https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart)
 2. Copy the example DAG into the `dags/` folder of your astro project
-3. Add the following env var to your `Dockerfile` to enable xcom pickling: `ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True`
-4. Copy the directories in the `include` folder of this repository into the `include` directory of your Astro project
-5. Add `airflow-provider-great-expectations` to your `requirements.txt`
+3. Copy the directories in the `include` folder of this repository into the `include` directory of your Astro project
+4. Copy your GCP `credentials.json` file into the base directory of your Astro project
+5. Add the following to your `Dockerfile` to install the `airflow-provider-great-expectations` package, enable xcom pickling, and add the required Airflow variables and connection to run the example DAG:
+   ```
+   RUN pip install --user airflow_provider_great_expectations
+   ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
+   ENV GOOGLE_APPLICATION_CREDENTIALS=/usr/local/airflow/credentials.json
+   ENV AIRFLOW_VAR_MY_PROJECT=<YOUR_GCP_PROJECT_ID>
+   ENV AIRFLOW_VAR_MY_BUCKET=<YOUR_GCS_BUCKET>
+   ENV AIRFLOW_VAR_MY_DATASET=<YOUR_BQ_DATASET>
+   ENV AIRFLOW_VAR_MY_TABLE=<YOUR_BQ_TABLE>
+   ENV AIRFLOW_CONN_MY_BIGQUERY_CONN_ID='google-cloud-platform://?extra__google_cloud_platform__scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fbigquery&extra__google_cloud_platform__project=bombora-dev&extra__google_cloud_platform__key_path=%2Fusr%2Flocal%2Fairflow%2Fairflow-gcp.bombora-dev.iam.gserviceaccount.com.json'
+   ```
 6. Run `astro dev start` to view the DAG on a local Airflow instance (you will need Docker running)
 
 **With a vanilla Airflow installation**:
 1. Add the example DAG to your `dags/` folder
 2. Make the `great_expectations` and `data` directories in `include/` available in your environment.
 3. Change the `data_file` and `ge_root_dir` paths in your DAG file to point to the appropriate places.
-4. Change the paths in `great-expectations/checkpoints/*.yml` to point to the absolute path of your data files. 
+4. Change the paths in `great-expectations/checkpoints/*.yml` to point to the absolute path of your data files.
+5. Change the value of [`enable_xcom_pickling`](https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg#L181) to `true` in your airflow.cfg
```

Comment:
Given that the instructions for Astronomer require copying stuff over to an Astronomer project directory, what would you think about having another directory that we could point to for a vanilla Airflow run (with the directories wired up to work)?

Reply:
Ah, yeah, that makes sense.  When you say "directory", you mean another `great-expectations` directory called, for instance, `great-expectations-vanilla-airflow`, with the appropriate absolute paths for a typical vanilla AF installation, I assume?

Reply:
Exactly. Maybe have a structure sort of like this

```
airflow-provider-great-expectations
 -> examples
   -> astronomer
     -> include
       -> ...  
   -> vanilla_airflow
     -> ... 
```